.github/workflows/code_quality_windows.yml

```diff
@@ -0,0 +1,28 @@
+# # This workflow will install Python dependencies, run tests and lint with a single version of Python
+# # For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+# name: Python application
+
+# on:
+#   pull_request:
+#     branches: [ main ]
+
+# jobs:
+#   build:
+
+#     runs-on: windows-latest
+
+#     steps:
+#     - uses: actions/checkout@v2
+#     - name: Set up Python 3.8
+#       uses: actions/setup-python@v2
+#       with:
+#         python-version: "3.8"
+#     - name: Install dependencies
+#       run: |
+#         $CONDA/bin/conda install -c conda-forge --file requirements.txt
+#         $CONDA/bin/python -m pip install --upgrade pip
+#         $CONDA/bin/pip install ".[dev]"
+#     - name: Test with pytest
+#       run: |
+#         $CONDA/bin/pytest --cov=openspoor --cov-report=xml --cov-fail-under=50
```

Comment:
I'm not sure if the code coverage is over 50% now, but we'll cross that bridge once we get there.

Reply:
it is 80ish